core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/Blueprint.scala

```diff
@@ -76,22 +76,15 @@ object Blueprint {
 
         val topics = if (config.hasPath(TopicsSectionKey)) {
           getKeys(config, TopicsSectionKey).map { key ⇒
-            val topicKey            = s"$TopicsSectionKey.${key}.$TopicKey"
-            val bootstrapServersKey = s"$TopicsSectionKey.${key}.$BootstrapServersKey"
-            val createKey           = s"$TopicsSectionKey.${key}.$ManagedKey"
-            val producersKey        = s"$TopicsSectionKey.${key}.$ProducersKey"
-            val consumersKey        = s"$TopicsSectionKey.${key}.$ConsumersKey"
-            val configKey           = s"$TopicsSectionKey.${key}"
-
-            val topic                    = getStringOrNone(config, topicKey).getOrElse(key)
-            val bootstrapServersOverride = getStringOrNone(config, bootstrapServersKey)
-            val create                   = getBooleanOrNone(config, createKey).getOrElse(true)
-            val producers                = getStringListOrEmpty(config, producersKey)
-            val consumers                = getStringListOrEmpty(config, consumersKey)
+            val topicKey     = s"$TopicsSectionKey.${key}.$TopicKey"
+            val producersKey = s"$TopicsSectionKey.${key}.$ProducersKey"
+            val consumersKey = s"$TopicsSectionKey.${key}.$ConsumersKey"
+            val configKey    = s"$TopicsSectionKey.${key}"
```

Comment:
@RayRoestenburg Perhaps I am forgetting - is the `topics` section mandatory - I mean even for managed configurations (`managed = yes`) ?

Reply:
This points to inside a topic, so `topics.my-topic` for instance, and nothing in there is mandatory. the `topics` section is allowed to be empty. (depending on the streamlets, this will result in an error showing that the streamlets inlets / outlets are not connected.)